parsely/parselyandroid/ParselyMetadata.java

```diff
@@ -5,51 +5,66 @@
 import java.util.HashMap;
 import java.util.Map;
 
-/*! \brief Represents article/video metadata to be passed to Parsely tracking.
+/*! \brief Represents article metadata to be passed to Parsely tracking.
  *
- *  This class is used to attach a metadata block to a Parse.ly pageview or video
- *  request. For most use cases this is only required for video data. Pageviews typically
- *  correspond to a URL which we can crawl on the customer site.
+ *  This class is used to attach a metadata block to a Parse.ly pageview
+ *  request. Pageview metadata is only required for URLs not accessible over the
+ *  internet (i.e. app-only content) or if the customer is using an "in-pixel" integration.
+ *  Otherwise, metadata will be gathered by Parse.ly's crawling infrastructure.
  */
 public class ParselyMetadata {
     public ArrayList<String> authors, tags;
-    public String canonical_url, section, thumbUrl, title;
+    public String canonicalUrl, section, thumbUrl, title;
     public Calendar pubDate;
 
+    /* \brief Create a new ParselyMetadata object.
+     *
+     * @param authors         List of authors for the article.
+     * @param canonicalUrl    Canonical URL of the article.
+     * @param section         Section of the article.
+     * @param tags            List of tags for the article.
+     * @param thumbUrl        URL of a thumbnail for the article.
+     * @param title           Title of the article.
+     * @param pubDate         Publish date of the article.
+     */
     public ParselyMetadata(
             ArrayList<String> authors,
-            String canonical_url,
+            String canonicalUrl,
             String section,
             ArrayList<String> tags,
             String thumbUrl,
             String title,
             Calendar pubDate
     ) {
         this.authors = authors;
-        this.canonical_url = canonical_url;
+        this.canonicalUrl = canonicalUrl;
         this.section = section;
         this.tags = tags;
         this.thumbUrl = thumbUrl;
         this.title = title;
         this.pubDate = pubDate;
     }
 
+    /* \brief Turn the ParselyVideoMetadata into a Map
```

Comment:
I'm not a fan of using the class name in the documentation of one of its members.
> Turn this object into a Map

Also, the class in this file is `ParselyMetadata`, not `ParselyVideoMetadata`.

Reply:
Woopsie there. I'll change both things about it.